### PR TITLE
Added 4 more GHSA sync'ed advisories

### PR DESCRIPTION
**Pull request by jasnow**
Added 4 more GHSA sync'ed advisories. Ran yamllint and rake (both green).
 *  gems/grit/CVE-2013-4489.yml
 * gems/openssl/CVE-2018-16395.yml
 * gems/passenger/CVE-2018-12027.yml
 * gems/passenger/CVE-2018-12028.yml

### CODE REVIEW

File: gems/grit/CVE-2013-4489.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  GitLab code search feature.
+
+  GitLab Grit Gem for Ruby contains a flaw in the app/contexts/search_context.rb
+  script. The issue is triggered when input passed via the code
+  search box is not properly sanitized, which allows strings to
+  be evaluated by the shell. This may allow a remote attacker to
+  execute arbitrary commands.
+cvss_v2: 6.5
+patched_versions:
+  - ">= 2.6.1"
```
Comment (postmodern) [inline gems/grit/CVE-2013-4489.yml:22]: I think this GHSA advisory is wrong. The [grit gem](https://rubygems.org/gems/grit) has no version 2.6.1. However, the [gitlab-grit gem](https://github.com/gems/gitlab-grit) gem does have a 2.6.1 version, and it's [ChangeLog mentions security fixes in 2.6.1](https://gitlab.com/gitlab-org/gitlab-grit/-/blob/v2.6.1/History.txt?ref_type=tags#L2). I've submitted a fix to the GHSA advisory.

Upon further investigation, it appears we already have a `gems/gitlab-grit/CVE-2013-4489.yml` file. So this is probably a duplicate.

File: gems/openssl/CVE-2018-16395.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  objects are compared using ==, depending on the ordering, non-equal objects may
+  return true. When the first argument is one character longer than the second, or
+  the second argument contains a character that is one less than a character in the
+  same position of the first argument, the result of == will be true. This could be
+  leveraged to create an illegitimate certificate that may be accepted as legitimate
+  and then used in signing or encryption operations.
+cvss_v2: 7.5
+cvss_v3: 9.8
+patched_versions:
+  - ">= 2.0.9"
```
Comment (postmodern) [inline gems/openssl/CVE-2018-16395.yml:19]: The URL mentions that the openssl gem 2.1.2 was released with the security fix. Should be `>= 2.1.2`.

I've also submitted a correction to the GHSA advisory.

**Review by postmodern — CHANGES_REQUESTED** [review]
* I think `gems/grit/CVE-2013-4489.yml` is a dup of `gems/gitlab-grit/CVE-2013-4489.yml` and can safely be removed. I also believe the upstream GHSA info is incorrect, and submitted a correction.
* I believe the `patched_versions` is incorrect for `gems/openssl/CVE-2018-16395.yml`.

**Comment by jasnow** [comment]
Too much bad, closing PR.

File: gems/grit/CVE-2013-4489.yml (hunk 1 above)
Comment (jasnow) [reply gems/grit/CVE-2013-4489.yml:22]: Delete as dup

File: gems/openssl/CVE-2018-16395.yml (hunk 2 above)
Comment (jasnow) [reply gems/openssl/CVE-2018-16395.yml:19]: Changed patched_version number.

**Comment by jasnow** [comment]
Overreacted, reopened PR and did fixes per your feedback.

**Comment by jasnow** [comment]
thanks

**Pull request merged**